dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -154,9 +154,9 @@ error.spring4.required.explanation=The currently configured Spring Boot feature
 error.spring4.required.useraction=Configure a version of the Spring Boot feature that is compatible with application content.
 
 #ADDED for SpringBoot 4.0:
-error.spring4.requires.servlet60.or.later.application=CWWKC0277E: The servlet-6.0 or later feature must be configured. The springBoot-4.0 feature is configured and the application uses servlets.
-error.spring4.requires.servlet60.or.later.application.explanation=The application uses servlets but no Servlet feature is configured.
-error.spring4.requires.servlet60.or.later.application.useraction=Configure the specified Servlet feature.
+error.spring4.requires.servlet61.or.later.application=CWWKC0277E: The servlet-6.1 or later feature must be configured. The springBoot-4.0 feature is configured and the application uses servlets.
```

Comment:
FATSuite and server.xml in io.openliberty.springboot.fat40_fat needs to be updated as well.

Reply:
We check for the error code in the FAT so it won't be a problem.

Reply:
I might just have picked a bad place to put the comment.